src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -270,12 +277,12 @@ bool AsyncRPCOperation_sendmany::main_impl() {
     }
 
     LogPrint("zrpc", "%s: spending %s to send %s with fee %s\n",
```

Comment:
Filed a PR where we can discuss and modify for 1.0.6 release: https://github.com/zcash/zcash/issues/2060